packages/bot-web-ui/src/components/transactions/transactions.tsx

```diff
@@ -104,6 +107,7 @@ const Transactions = observer(({ is_drawer_open }) => {
                                 }
                             }}
                             getRowSize={({ index }) => {
+                                // console.log('index', index);
```

Comment:
can we remove this console? @farabi-deriv

Reply:
Missed this one. Thanks for catching this. @sandeep-deriv